CONTRIBUTING.md

```diff
@@ -178,6 +192,12 @@ contains the App Service, App Service Plan and SignalR resources.
 CollaborationStateServiceUrl value to the App Service URL -
 `https://<RESOURCE_NAME_SUFFIX>.azurewebsites.net` and redeploy the extension.
 
+#### Considerations for the Backend
```

Comment:
@mindlessroman I think it's safe to remove this "Considerations" section since you incorporated the details above.